go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/cosmos/cosmos-db
 go 1.19
 
 require (
-	github.com/cockroachdb/pebble v1.1.0
-	github.com/google/btree v1.1.2
+	github.com/cockroachdb/pebble v1.1.2
+	github.com/google/btree v1.1.3
 	github.com/linxGnu/grocksdb v1.8.12
```

Comment:
cometbft-db bumped this, we may not want to do that in the same release line as it may break stuff.